examples/profile.js

```diff
@@ -21,16 +34,92 @@ app.module = angular.module('app', ['ngeo']);
  */
 app.MainController = function($http, $scope) {
 
+  this['profile'] = null;
```

Comment:
It seems unusual to have the object set to null in the controller and have it instanciated by the directive.